test/sphere_metric_terms.jl

```diff
@@ -18,3 +17,33 @@ using Test
 
     @test sum(ones(space)) ≈ 4pi * radius^2 rtol = 1e-3
 end
+
+@testset "Volume of a sphere" begin
+    FT = Float64
+    radius = FT(3)
+    zlim = (0, 1)
+    helem = 4
+    zelem = 10
+    Nq = 4
+
+    vertdomain = Domains.IntervalDomain(
+        Geometry.ZPoint{FT}(zlim[1]),
+        Geometry.ZPoint{FT}(zlim[2]);
+        boundary_tags = (:bottom, :top),
+    )
+    vertmesh = Meshes.IntervalMesh(vertdomain, nelems = zelem)
+    vert_center_space = Spaces.CenterFiniteDifferenceSpace(vertmesh)
+
+    horzdomain = Domains.SphereDomain(radius)
+    horzmesh = Meshes.Mesh2D(horzdomain, Meshes.EquiangularSphereWarp(), helem)
+    horztopology = Topologies.Grid2DTopology(horzmesh)
+    quad = Spaces.Quadratures.GLL{Nq}()
+    horzspace = Spaces.SpectralElementSpace2D(horztopology, quad)
+
+    hv_center_space =
+        Spaces.ExtrudedFiniteDifferenceSpace(horzspace, vert_center_space)
+    hv_face_space = Spaces.FaceExtrudedFiniteDifferenceSpace(hv_center_space)
```

Comment:
do you need this `hv_face_space` here? It seems that for the sum below you only use the center space

Reply:
Thanks! I'll remove it.